src/laundry/pdf.clj

```diff
@@ -17,21 +17,28 @@
         (proxy-super close)
         (io/delete-file path)))))
 
+(s/defn validate-pdf-settings [pdfsettings :- s/Str]
+  (when-not (#{"/screen" "/ebook" "/printer" "/prepress" "/default"} pdfsettings) "pdfsettings is not given in correct format"))
```

Comment:
If you want, you could use the set of correct values for a friendlier error messages

Reply:
Now that this was merged, the values remain for the API user to check from Ghostscript documentation